Uncaught Type JavaScript Error In Uno.Lottie

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

If the matter is security related, please disclose it privately via https://github.com/nventive/Uno/security/
-->

## Current behavior

<!-- Describe how the issue manifests. -->

Hi,
I have an AnimatedVisualPlayer on my Page for playing a Lottie:

```
<winui:AnimatedVisualPlayer
    x:Name="player"
    AutoPlay="true" Width="150" Height ="150" >

    <lottie:LottieVisualSource
        UriSource="ms-appx:///Lottie/waiter.json" />
</winui:AnimatedVisualPlayer>
```

I installed these packages:

- Microsoft.Toolki.Uwp.UI.Lottie (for the LottieVisualSource)
- Microsoft.UI.Xaml (for the AnimatedVisualPlayer)
- Uno.UI.Lottie (for the LottieVisualSource)
- Uno.UI (for the AnimatedVisualPlayer)

I did it also:

> On WASM, iOS and macOS, you can put the Lottie .json files directly in a folder of the shared project (for example "Lottie/myanimation.json") and set their Build action as Content.


**UWP can play the Lottie seamlessly:**

![AnimatedVisualPlayerLottie_UWP](https://user-images.githubusercontent.com/23173536/82942813-4090bd80-9fa1-11ea-9c3a-26adbd0b305e.png)

**However, WASM cannot play Lottie with this error:**

![AnimatedVisualPlayerLottie_WASM](https://user-images.githubusercontent.com/23173536/82942946-7766d380-9fa1-11ea-8deb-acb95db161f6.png)

![UnoLottieJS55](https://user-images.githubusercontent.com/23173536/82942958-7d5cb480-9fa1-11ea-89d5-fbc026bb732e.png)

**The error log is:** 
[localhost-1590518978914.log](https://github.com/unoplatform/uno/files/4684612/localhost-1590518978914.log)

**My custom lottie is:**

[waiter.json.zip](https://github.com/unoplatform/uno/files/4684622/waiter.json.zip)


## Expected behavior

WASM can play lotties based on this document: https://platform.uno/docs/articles/features/Lottie.html

AND
this page:

https://lottie.platform.uno/
## How to reproduce it (as minimally and precisely as possible)

<!-- Please provide a **MINIMAL REPRO PROJECT** and the **STEPS TO REPRODUCE**-->
Create a AnimatedVisualPlayer and add my lottie. After that, compare UWP app with WASM.
## Environment

<!-- For bug reports Check one or more of the following options with "x" -->

Nuget Package:

Package Version(s):
Uno.UI.RemoteControl {2.4.0}
Newtonsoft.Json {12.0.3}
Uno.UI.Lottie {2.4.0}
Uno.Wasm.Bootstrap {1.2.0}
Uno.Wasm.Bootstrap.DevServer {1.2.0}
ACM_Search_AdminApps.Shared {1.1.0}
Microsoft.Extensions.Logging.Filter {1.1.2}
Microsoft.Extensions.Logging.Con... {1.1.1}
NETStandard.Library {2.0.3}
Uno.UI {2.4.0}
ACM_Search_AdminApps.Shared {1.1.0}
Microsoft.NETCore.UniversalWindo... {6.2.10}
Microsoft.Extensions.Logging.Con... {1.1.1}
Microsoft.Extensions.Logging.Filter {1.1.2}
Microsoft.Toolkit.Uwp.UI.Lottie {6.0.0}
Microsoft.UI.Xaml {2.4.2}
Newtonsoft.Json {12.0.3}
Uno.Core {2.0.0}
Microsoft.Toolkit.Uwp.UI.Lottie {6.0.0}

Affected platform(s):

- [ ] iOS
- [ ] Android
- [X] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [X] 2019 (version: 16.6 )
- [ ] for Mac (version: )

Relevant plugins:

- [X] Resharper (version: 2020.1.3 )

## Anything else we need to know?

<!-- We would love to know of any friction, apart from knowledge, that prevented you from sending in a pull-request -->

Errors: 

- "TypeError: Cannot read property 'animation' of undefined" executing javascript: "Uno.UI.Lottie.pause(403458);"

- dotnet.js:1 Error #1 "SyntaxError: Unexpected token '}'" executing javascript: "Uno.UI.Lottie.setAnimationProperties({elementId:282194,jsonPath:"/Lottie/waiter.json",autoplay:true,stretch:"Uniform",rate:1,5});" . 

[localhost-1590594616979.log](https://github.com/unoplatform/uno/files/4689924/localhost-1590594616979.log)

Maybe a little "}" causes a problem  @carldebilly ? I added you to a template project on Github after StackOverFlow comment: https://stackoverflow.com/questions/62030326/uncaught-type-javascript-error-in-uno-lottie-webassembly. Thanks for support.

--- a/src/AddIns/Uno.UI.Lottie/LottieVisualSource.wasm.cs
+++ b/src/AddIns/Uno.UI.Lottie/LottieVisualSource.wasm.cs
@@ -57,7 +57,7 @@ namespace Microsoft.Toolkit.Uwp.UI.Lottie
 					",stretch:\"",
 					player.Stretch.ToString(),
 					"\",rate:",
-					player.PlaybackRate.ToString(),
+					player.PlaybackRate.ToStringInvariant(),
 					"},",
 					jsonString,
 					");"

